pkg/scheduler/schedulercache/node_info.go

```diff
@@ -74,6 +78,14 @@ func initializeNodeTransientInfo() nodeTransientInfo {
 	return nodeTransientInfo{AllocatableVolumesCount: 0, RequestedVolumes: 0}
 }
 
+// nextGeneration: Let's make sure history never forgets the name...
+// Increments the generation number monotonically ensuring that generation numbers never collide.
+// Collision of the generation numbers would be particularly problematic if a node was deleted and
+// added back with the same name. See issue#63262.
+func nextGeneration() int64 {
+	return atomic.AddInt64(&generation, 1)
```

Comment:
Wondering what will happen if overflow occurs. I quickly looked up if `Add` could panic, seems it doesn't in all cases.

Reply:
Check this upstream issue: https://github.com/golang/go/issues/20687

Maybe we can leave a comment with "Overflow not handled" :)

Reply:
If the counter is incremented 1 billion times per second, it will take over 200 years for it to overflow. Should we be really worry about its overflowing?! :-)

Reply:
agree. int64 won't overflow.

Reply:
Just commented for fun, overflow should not be the concern :-)

Reply:
Ack :)